.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
Use this to make sure everything in node modules is excluded 
`node_modules/* `